ts/components/FiscalCodeComponent.tsx

```diff
@@ -221,6 +304,34 @@ const styles = StyleSheet.create({
 });
 
 export default class FiscalCodeComponent extends React.Component<Props> {
+  private getGender(fiscalCode: FiscalCode) {
```

Comment:
I suppose all these functions used to extract info from a fiscal code can be used again
Could you place them into a utils file?
Perhaps you can expose a single function that returns something like this from a given fiscal code:

```javascript
type DerivedFiscalCodeData = {
  sex: "M" | "F";
  birthDate: Date;
  codiceCatastale: string;
};
```
NS: codiceCatastale will have its own type